src/components/Menu/Menu.tsx

```diff
@@ -1,4 +1,4 @@
-import React, {useState, useEffect} from 'react';
+import React, {useState, useEffect, ReactElement} from 'react';
```

Comment:
As you use `React.ReactElement`, you don't need this import.
```suggestion
import React, {useState, useEffect} from 'react';
```